modules/lib/src/main/resources/assets/js/app/inputtype/customselector/CustomSelectorComboBox.ts

```diff
@@ -15,6 +15,18 @@ export class CustomSelectorComboBox
         super(builder);
     }
 
+    protected reload(inputValue: string): Q.Promise<any> {
+        if (this.getLoader().isLoaded() && this.getLoader().getSearchString() === inputValue) {
+            return this.getLoader().search(inputValue);
+        }
+
+        this.getLoader().setSearchString(inputValue);
+
+        return this.getLoader().load().then(() => {
+            return this.getLoader().search(inputValue);
+        });
```

Comment:
Can we replace `this.getLoader()` with a `const` please?

Reply:
Done